src/sunsynk/definitions3phhv.py

```diff
@@ -0,0 +1,400 @@
+    Sensor(618, "Grid CT L3 power", WATT, -1),  # cPower
+)
+
+#############
+# Load Power
+#############
+SENSORS += (
+    Sensor(653, "Load power", WATT, -1),
+    Sensor(650, "Load L1 power", WATT, -1),
+    Sensor(651, "Load L2 power", WATT, -1),
+    Sensor(652, "Load L3 power", WATT, -1),
+)
+
+#
+################
+# Solar Power
+################
+SENSORS += (
+    MathSensor((672, 673), "PV power", WATT, factors=(10, 10)),  # pv1 power,pv2 power
+)
+################
+# MPPT 1 Solar Power
+################
+SENSORS += (
+    Sensor(672, "PV1 power", WATT, 10),
+    Sensor(676, "PV1 voltage", VOLT, 0.1),
+    Sensor(677, "PV1 current", AMPS, 0.1),
+)
+
+################
+# MPPT2 Solar Power
+################
+SENSORS += (
+    Sensor(673, "PV2 power", WATT, 10),
+    Sensor(678, "PV2 voltage", VOLT, 0.1),
+    Sensor(679, "PV2 current", AMPS, 0.1),
+)
+
+################
+# MPPT3 Solar Power
+################
+SENSORS += (
+    Sensor(674, "PV3 power", WATT, 10),
+    Sensor(680, "PV3 voltage", VOLT, 0.1),
+    Sensor(681, "PV3 current", AMPS, 0.1),
+)
+
+################
+# MPPT4 Solar Power
+################
+SENSORS += (
+    Sensor(675, "PV4 power", WATT, 10),
+    Sensor(682, "PV4 voltage", VOLT, 0.1),
+    Sensor(683, "PV4 current", AMPS, 0.1),
+)
+
+
+###################
+# Power on Outputs (aka GEN - which has multiple modes on the deye 3 Phase )
+###################
+SENSORS += (
+    Sensor(667, "Gen power", WATT, -1),
+    Sensor(664, "Gen L1 power", WATT, -1),
+    Sensor(665, "Gen L2 power", WATT, -1),
+    Sensor(666, "Gen L3 power", WATT, -1),
+)
+
+###################
+# Energy
+###################
+SENSORS += (
+    Sensor(502, "Day Active Energy", KWH, -0.1),
+    Sensor(514, "Day Battery Charge", KWH, 0.1),
+    Sensor(515, "Day Battery discharge", KWH, 0.1),
+    Sensor(521, "Day Grid Export", KWH, 0.1),
+    Sensor(520, "Day Grid Import", KWH, 0.1),
+    Sensor(536, "Day Gen Energy", KWH, 0.1),
+    Sensor(526, "Day Load Energy", KWH, 0.1),  # I guess "used" = load?
+    Sensor(529, "Day PV Energy", KWH, 0.1),
+    Sensor((506, 507), "Total Active Energy", KWH, 0.1),  # signed?
+    Sensor((516, 517), "Total Battery Charge", KWH, 0.1),
+    Sensor((518, 519), "Total Battery Discharge", KWH, 0.1),
+    Sensor((524, 525), "Total Grid Export", KWH, 0.1),
+    Sensor((522, 523), "Total Grid Import", KWH, 0.1),
+    Sensor((527, 528), "Total Load Energy", KWH, 0.1),
+    Sensor((534, 535), "Total PV Energy", KWH, 0.1),
+)
+
+##########
+# General
+##########
+RATED_POWER = Sensor((20, 21), "Rated power", WATT, 0.1)
+SENSORS += RATED_POWER
+
+SENSORS += (
+    Sensor(
+        0, "Device Type"
+    ),  # {2: "Inverter", 3: "Hybrid Inverter", 4: "Micro Inverter", 5: "3 Phase Hybrid Inverter" }),
+    FaultSensor((555, 556, 557, 558), "Fault"),
+    InverterStateSensor(500, "Overall state"),
+    SDStatusSensor(0, "SD Status", ""),  # type: ignore        # 3 Phase does not have SD Card but crashes when removed
```

Comment:
```suggestion
    SDStatusSensor(0, "SD Status", ""),  # type: ignore
```
Not sure why this would crash? Seems like a leftover from the 3-ph